### PR TITLE
Added real world custom pagination example

### PR DESCRIPTION
**Pull request by jifeon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,6 +350,40 @@ parser.addPagination('custom-pagination', function (options) {
...
+Use it
+```js
+var pagination = {
+  type: 'clickPerPage', // your custom type
+  pageScope: '.page__content',
+  scope: '.page__pagination'
+};
+
+var parser = new Parser({
+    rules: {},
```
Comment (maZahaca) [inline README.md]: As I know that must be passed to parse method

Comment (jifeon) [inline README.md]: fixed

Comment (maZahaca) [inline README.md]: Again missed.

Take a look on this https://github.com/redco/goose-parser/blob/master/tests/phantom_parser_test.js#L301

For some reason we pass pagination on `Parser` create.
But rules and actions on the `parser.parse()`

Comment (jifeon) [inline README.md]: argh...

sooo hard )))

Comment (jifeon) [inline README.md]: new attempt! :boar:

**Pull request merged**